test/bootloader_base_test.rb

```diff
@@ -14,6 +19,14 @@
 
       subject.write
     end
+
+    context "Mode.normal is set" do
+      it "install packages listed required by bootloader from packages method" do
```

Comment:
Hardly understandable to me, probably simple `installs packages required by bootloader` would be enough.